Source/Core/Core/Boot/Boot.cpp

```diff
@@ -593,3 +588,38 @@ void CreateSystemMenuTitleDirs()
   const auto es = IOS::HLE::GetIOS()->GetES();
   es->CreateTitleDirectories(Titles::SYSTEM_MENU, IOS::SYSMENU_GID);
 }
+
+void RAMOverrideForIPLMemoryValues(std::string& data, const u32 ipl_hash)
+{
+  if (Config::Get(Config::MAIN_RAM_OVERRIDE_ENABLE))
```

Comment:
Now that this is localized within its own function, this can be inverted like so to allow unindenting all of the code within it.

```cpp
if (!Config::Get(Config::MAIN_RAM_OVERRIDE_ENABLE))
  return;
```